docs/docs/debugging-the-build-process.md

```diff
@@ -40,6 +41,33 @@ TypeError: Cannot read property 'internal' of undefined
     D:/dev/blog-v2/gatsby-node.js:6:12
 ```
 
+## Debugging with Node.js' built-in console
+
+One of the fastest ways to gain insight into Gatsby's build process is using the `console` functionality [built into Node.js](https://nodejs.org/en/knowledge/getting-started/the-console-module/). This works similar to how you might be used to in the browser.
+
+Adding a console.log statement in our sample from above will print the variable into your terminal. There you might notice that `args` contains a lower-cased node variable.
```

Comment:
```suggestion
Adding a `console.log` statement in the sample from above will print the variable into your terminal. There you might notice that `args` contains a lower-cased node variable.
```

Updating to adhere to the Gatsby Style Guide: https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-you-as-the-pronoun